src/renderer/components/poolActionsHistory/PoolActionsHistoryList.tsx

```diff
@@ -33,8 +33,15 @@ const renderItem = (goToTx: (txId: string) => void) => (action: PoolAction) => {
       )}
     </>
   )
+
+  const rowKey = FP.pipe(
+    oTxId,
+    O.map((id) => id),
+    O.getOrElse(() => action.date.toString())
```

Comment:
`${action.date.toString()}-{Math.random()}` - to make it little bit more unique ...